discovery/linode/linode.go

```diff
@@ -276,17 +304,23 @@ func (d *Discovery) refreshData(ctx context.Context) ([]*targetgroup.Group, erro
 		}
 
 		if instance.IPv6 != "" {
+			slaac := strings.Split(instance.IPv6, "/")[0]
 			for _, detailedIP := range detailedIPs {
-				if detailedIP.Address != strings.Split(instance.IPv6, "/")[0] {
+				if detailedIP.Address != slaac {
 					continue
 				}
-
 				publicIPv6 = detailedIP.Address
 
 				if detailedIP.RDNS != "" && detailedIP.RDNS != "null" {
 					publicIPv6RDNS = detailedIP.RDNS
 				}
 			}
+			for _, ipv6Range := range ipv6RangeList {
+				if ipv6Range.RouteTarget == slaac {
+					ip6cidr := ipv6Range.Range + "/" + strconv.Itoa(ipv6Range.Prefix)
+					ipv6Ranges = append(ipv6Ranges, ip6cidr)
```

Comment:
```suggestion
					ipv6Ranges = append(ipv6Ranges, ipv6Range.Range + "/" + strconv.Itoa(ipv6Range.Prefix))
```

This seems concise enough to inline (though I think I'd vote in favor of `fmt.Sprintf()` here)

Reply:
The only reason i used strconv.Itoa vs fmt.Sprintf is due to a note on performance.  I'm happy to use fmt.Sprintf if that's the expected style.

`fmt.Sprintf, strconv.Itoa and strconv.FormatInt will do the job. But Sprintf will use the package reflect, and it will allocate one more object, so it's not an efficient choice.`

Reply:
Pivoted to fmt.Sprintf() in 4c66a179ae5da10632066ab6e3a60943c71699bf